qemu/qmp.go

```diff
@@ -670,7 +670,7 @@ func (q *QMP) ExecuteBlockdevAdd(ctx context.Context, device, blockdevID string)
 // to a previous call to ExecuteBlockdevAdd.  devID is the id of the device to
 // add.  Both strings must be valid QMP identifiers.  driver is the name of the
 // driver,e.g., virtio-blk-pci, and bus is the name of the bus.  bus is optional.
```

Comment:
The documentation doesn't mention the new parameter.

Reply:
@markdryan I have addresses this. PTAL.